tests/torchtune/models/qwen2/test_qwen2.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
I'm sure @ebsmothers will have more thoughts on whether this is necessary here - but I'll share an excellent comment from him about the testing process in torchtune https://github.com/pytorch/torchtune/pull/840#discussion_r1579801172.